commit d287698c43d7915e422d298985891eb609b511b8
Author: Ludovic Desroches <ludovic.desroches@atmel.com>
Date:   Fri Mar 11 11:43:39 2016 +0100

    ARM: dts: at91: sama5d3 Xplained: don't disable hsmci regulator
    
    commit ae3fc8ea08e405682f1fa959f94b6e4126afbc1b upstream.
    
    If enabling the hsmci regulator on card detection, the board can reboot
    on sd card insertion. Keeping the regulator always enabled fixes this
    issue.
    
    Signed-off-by: Ludovic Desroches <ludovic.desroches@atmel.com>
    Fixes: 1b53e3416dd0 ("ARM: at91/dt: sama5d3 xplained: add fixed regulator for vmmc0")
    Signed-off-by: Nicolas Ferre <nicolas.ferre@atmel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/arch/arm/boot/dts/at91-sama5d3_xplained.dts b/arch/arm/boot/dts/at91-sama5d3_xplained.dts
index ff888d2..f3e2b96 100644
--- a/arch/arm/boot/dts/at91-sama5d3_xplained.dts
+++ b/arch/arm/boot/dts/at91-sama5d3_xplained.dts
@@ -303,6 +303,7 @@
 		regulator-name = "mmc0-card-supply";
 		regulator-min-microvolt = <3300000>;
 		regulator-max-microvolt = <3300000>;
+		regulator-always-on;
 	};
 
 	gpio_keys {
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1428_d287698c43d7915e422d298985891eb609b511b8.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1428_d287698c43d7915e422d298985891eb609b511b8.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
